README.md

```diff
@@ -71,6 +73,10 @@ WD-Firmware
       $ sudo ln -s /usr/lib/x86_64-linux-gnu/libmpfr.so.6 /usr/lib/x86_64-linux-gnu/libmpfr.so.4 
 - Download and install Java SE Runtime Environment
 
+- For RISC-V OpenOCD, you will need to following depended libs: libusb-0.1, libusb-1.0-0-dev, libusb-dev

```

Comment:
"you will need **the** following", not "you will need **to** following"

Reply:
done